App crash after tapping on Downloads from Library

### Steps to reproduce

1. Tap on Library
2. Tap on Downloads

### Expected behavior

I suppose the app should be showing videos that are stored on the device.

### Actual behavior

Crash to desktop. A crash report is shown after the app is restarted.

### LibreTube version

0.7.0

### Android version

Android 11

### Other details

```
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.reflect.InvocationTargetException
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
... 1 more
Caused by: java.io.FileNotFoundException: /storage/emulated/0/Android/data/com.github.libretube/files/thumbnail/#Shorts Meloni sbaglia il nome di Soumahoro e gli dà del tu. La reazione dei colleghi di partito: open failed: ENOENT (No such file or directory)
at libcore.io.IoBridge.open(IoBridge.java:492)
at java.io.FileInputStream.<init>(FileInputStream.java:160)
at java.io.FileInputStream.<init>(FileInputStream.java:115)
at android.content.ContentResolver.openInputStream(ContentResolver.java:1473)
at com.github.libretube.ui.fragments.DownloadsFragment.onViewCreated(DownloadsFragment.kt:440)
at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:305)
at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:154)
at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:1085)
at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:92)
at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:74)
at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:4)
at android.os.Handler.handleCallback(Handler.java:938)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:223)
at android.app.ActivityThread.main(ActivityThread.java:7664)
... 3 more
Caused by: android.system.ErrnoException: open failed: ENOENT (No such file or directory)
at libcore.io.Linux.open(Native Method)
at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
at libcore.io.BlockGuardOs.open(BlockGuardOs.java:254)
at libcore.io.ForwardingOs.open(ForwardingOs.java:166)
at android.app.ActivityThread$AndroidOs.open(ActivityThread.java:7550)
at libcore.io.IoBridge.open(IoBridge.java:478)
... 17 more 
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

--- a/app/src/main/java/com/github/libretube/util/ImageHelper.kt
+++ b/app/src/main/java/com/github/libretube/util/ImageHelper.kt
@@ -72,6 +72,7 @@ object ImageHelper {
             DownloadHelper.getDownloadDir(context, DownloadHelper.THUMBNAIL_DIR),
             fileName
         )
+        if (!file.exists()) return null
         return getImage(context, Uri.fromFile(file))
     }
 

